contracts/contracts/sbtc-bootstrap-signers.clar

```diff
@@ -27,12 +28,20 @@
 ;; Rotate keys
 ;; Used to rotate the keys of the signers. This is called whenever
 ;; the signer set is updated.
-(define-public (rotate-keys-wrapper (new-keys (list 128 (buff 33))) (new-aggregate-pubkey (buff 33)))
+(define-public (rotate-keys-wrapper 
+    (new-keys (list 128 (buff 33)))
+    (new-aggregate-pubkey (buff 33))
+    (new-signature-threshold uint)
+  )
     (let 
         (
             (current-signer-data (contract-call? .sbtc-registry get-current-signer-data))   
-            (new-signer-principal (pubkeys-to-principal new-keys signature-threshold))
+            (new-signer-principal (pubkeys-to-principal new-keys new-signature-threshold))
         )
+        ;; Check that the signature threshold is valid
+        (asserts! (and (> new-signature-threshold (/ (len new-keys) u2))
```

Comment:
Where does this "rule" come from? If we're going to let the signers determine a new threshold (in this code, between 50%-100%), why not lower?

Reply:
Good questions. I think I originally had it at any positive threshold and thought "we'll probably want to ensure that the threshold will never be less than 50%". So, this comes out of thin air. Either greater than 50% or greater than 0 sound defensible to me and I do not have any strong feels on either.

Reply:
Yep, I think that's fair!